diesel_compile_tests/tests/compile-fail/empty_insertable_struct.rs

```diff
@@ -0,0 +1,20 @@
+#[macro_use]
+extern crate diesel;
+#[macro_use]
+extern crate diesel_codegen;
+
+use diesel::*;
+
+table! {
+    users {
+        id -> Integer,
+    }
+}
+
+#[derive(Insertable)]
+//~^ ERROR no rules expected the token `]`
```

Comment:
Yes, we should be panicking inside of codegen and never getting to the Rust macro